.travis.yml

```diff
@@ -16,7 +16,7 @@ script: scripts/travis-run.sh
 env:
   global:
     - SUBDAGS=1
-    - BIOCONDA_UTILS_TAG=master
+    - BIOCONDA_UTILS_TAG=415709197fc3390aecbb4ab61e90950b5a192da5
     - BIOCONDA_UTILS_BUILD_ARGS="--loglevel=info"
```

Comment:
This one: https://github.com/bioconda/bioconda-utils/commit/7a67a25088d5243dce8f662b779ae54fd41dd67b

The update to latest conda did not worked :(